internal/services/loganalytics/log_analytics_linked_service_resource_test.go

```diff
@@ -163,7 +163,14 @@ resource "azurerm_log_analytics_workspace" "test" {
   resource_group_name = azurerm_resource_group.test.name
   sku                 = "PerGB2018"
   retention_in_days   = 30
+
+  lifecycle {
+    ignore_changes = [
+      sku
+    ]
+  }
 }
```

Comment:
if this worked before this means you are going to affect peoples configurations?

How come this is needed now?

Reply:
Per the description in #17069, it used to make it not "force new", still changed, and acctest can not pass. and then, due to SDK upgraded, it becomes "force new" again. fix this workaround only to make it not "force new".

Reply:
again, we shouldn't need to ignore changes, its a sign something is wrong with the resource.

Reply:
As this seems to be a separate issue from the one being fixed in this PR, I'm going to remove this until we dedicate some more time to looking into it